### PR TITLE
Embedded forms

### PR DESCRIPTION
**Pull request by larkox**
#### Summary
Add Embedded forms

#### Ticket Link
None

#### Related Pull Requests
Apps: https://github.com/mattermost/mattermost-plugin-apps/pull/22
Webapp: https://github.com/mattermost/mattermost-webapp/pull/6804

### CODE REVIEW

File: src/actions/integrations.ts [hunk 1]
```diff
@@ -381,3 +381,25 @@ export function submitInteractiveDialog(submission: DialogSubmission): ActionFun
         return {data};
     };
 }
+
+export function submitEmbeddedForm(submission: DialogSubmission, postID: string, appID: string): ActionFunc {
+    return async (dispatch: DispatchFunc, getState: GetStateFunc) => {
+        const state = getState();
+        submission.channel_id = getCurrentChannelId(state);
+        submission.team_id = getCurrentTeamId(state);
+        submission.submission.mm_postID = postID;
```
Comment (mickmister) [inline src/actions/integrations.ts:390]: This seems weird to mix camelCase and snake_case. Also, why is the `mm_` prefix needed?

**Review by mickmister — CHANGES_REQUESTED** [review]
Looks good, just a question/request to be consistent with camelCase and snake_case

File: src/actions/integrations.ts (hunk 1 above)
Comment (larkox) [reply src/actions/integrations.ts:390]: To make sure it does not collide with any other possible value that is using the same namespace.
Will fix the weird casing.

**Comment by mickmister** [comment]
@larkox Should this PR be closed since we are using `doAppCall` now?

**Pull request closed without merging**